napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -268,6 +268,9 @@ def cancel(
             except KeyError:
                 pass
 
+    def help(self):
```

Comment:
@haesleinhuepf thanks for working on this. handle action is making assumptions, like it refers to install/uninstall/update like methods on the plugin installer.

Piggybacking on that method for the help case, which is different results in an error in another part of the code I would suggest removing that code and actually using this separate method to do things since it is completely unrelated to install methods.

Something like:

```python
def help(self, pkg_name)
            import webbrowser
            webbrowser.open("https://napari-hub.org/plugins/" + pkg_name)
```